08b_Interactive_Catalog_Visualization.ipynb

```diff
@@ -912,7 +623,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 4.2 Colour-Magnitude Diagram linked to a Skyplot\n",
+    "#### 3.2 Colour-Magnitude Diagram linked to a Skyplot\n",
     "\n",
     "We will use bokeh to plot a color-magnitude (g vs. g-i) diagram making use of the cModel magnitudes and a skyplot and link link the two"
    ]
```

Comment:
The points are small enough that I was actually having a bit of a hard time hovering over them. Once I did hover over them, I had a hard time finding the corresponding point that turned red in the other panel. Maybe we could make them a bit bigger?

Reply:
done